### PR TITLE
Monkey Emotes Fix

### PR DESCRIPTION
**Pull request by formlessnameless**
See title. I don't remember if they used to have bespoke sounds or not, but they can at least scream again. And other stuff. 

:cl:
- fix: Monkeys can scream properly again. Also other emotions.

### CODE REVIEW

**Comment by hivehum** [comment]
there is no monkey SoundCollection. im not sure where the monkey emote sounds went

**Comment by formlessnameless** [comment]
did monkeys ever have their own fucking noises. am i going crazy

**Comment by formlessnameless** [comment]
i think i gaslit myself about monkey sounds. oh well. now they at least scream with a sound like when they try to climb something. if anyone wants to add more monkey sounds the structure is there now at least

File: Resources/Prototypes/_Impstation/Voice/speech_emote_sounds.yml [hunk 1]
```diff
@@ -83,3 +83,18 @@
...
+  sounds:
+    Scream:
+      path: /Audio/Animals/monkey_scream.ogg
+      params:
+        variation: 0.125
+        pitch: 0.90
+    MonkeyScreeches:
+      collection: MonkeyScreeches
+    DefaultDeathgasp:
+      path: /Audio/_Impstation/Animals/bros_hurt.ogg
```
Comment (hivehum) [inline Resources/Prototypes/_Impstation/Voice/speech_emote_sounds.yml:100]: i see what you did here.

**Review by hivehum — APPROVED** [review]

**Pull request merged**